lib/countly_flutter.dart

```diff
@@ -1267,9 +1294,14 @@ class Countly {
   }
 
   /// Enable campaign attribution reporting to Countly.
-  /// For iOS use 'recordAttributionID' instead of 'enableAttribution'
+  /// Deprecated for iOS, for iOS use 'recordAttributionID' instead.
```

Comment:
It isn't deprecated "just for iOS", it's deprecated for all platforms

Reply:
but this change should be moved to the other PR and should not be part of this

Reply:
Implemented attribution changes in SDK-528 branch.